recipes/s3fs/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.0.4" %}
+
+package:
+  name: s3fs
+  version: "{{ version }}"
+
+source:
+  fn: s3fs-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/s3fs/s3fs-{{ version }}.tar.gz
+  md5: 4fa1bcec029283a904218544c1fa5eab
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
```

Comment:
Yeah, I have seen `setuptools` get installed anyways. I bet if we disable [`add_pip_as_python_depedency`]( https://github.com/conda/conda/blob/65d514ed043ed7dacaef60a9cef4ac731f3b11db/conda/config.py#L76 ) then this will stop happening. We probably should start doing that for other reasons. ( https://github.com/conda-forge/staged-recipes/pull/551 ) ( https://github.com/conda-forge/staged-recipes/pull/552 )